examples/simple/posix/UB-PTHREAD5-bad.c

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include<pthread.h>
+#include<stdlib.h>
+
+
+pthread_mutex_t * lock_ptr;
+
+
+int main(void)
+{
+    //Calling pthread_mutex_lock without a call to pthread_mutex_init
+    lock_ptr = malloc(sizeof(pthread_mutex_t));
```

Comment:
I think to make this more interesting what we should do is to initialize the mutex with pthread_mutex_init, but then copy it with memcpy and call the method on the copied mutex.